12-master/Dockerfile

```diff
@@ -0,0 +1,168 @@
+FROM postgres:12
+
+LABEL maintainer="PostGIS Project - https://postgis.net"
+
+ENV SFCGAL_VERSION master
+ENV SFCGAL_GIT_HASH b11af653fd84f5a4a959e4369019dd3962f904c3
+ENV PROJ_VERSION master
+ENV PROJ_GIT_HASH 341e0e7eb4f95de5c6ccc24ee98b7bb304964449
+ENV GDAL_VERSION master
+ENV GDAL_GIT_HASH 04a8d012807a2d2dd66085e68b167e3ade164348
+ENV GEOS_VERSION master
+ENV GEOS_GIT_HASH e37d04f8384b2e1fee83d64c6221adc3749a404a
+ENV POSTGIS_VERSION master
+ENV POSTGIS_GIT_HASH 8eecc746627f200f0df834ca19a3aea92db3782f
+
+RUN set -ex \
+    && apt-get update \
+    && apt-get install -y --no-install-recommends \
+      libboost-atomic1.67.0 \
+      libboost-chrono1.67.0 \
+      libboost-date-time1.67.0 \
+      libboost-filesystem1.67.0 \
+      libboost-program-options1.67.0 \
+      libboost-serialization1.67.0 \
+      libboost-system1.67.0 \
+      libboost-test1.67.0 \
+      libboost-thread1.67.0 \
+      libboost-timer1.67.0 \
+      libcgal13 \
+      libcurl3-gnutls \
+      libgmp10 \
+      libgmpxx4ldbl \
+      libjson-c3 \
+      libmpfr6 \
+      libprotobuf-c1 \
+      libtiff5 \
+      libxml2 \
+      sqlite3 \
+      curl \
+    && apt-get install -y --no-install-recommends \
+      autoconf2.13 \
+      automake \
+      autotools-dev \
+      bison \
+      build-essential \
+      ca-certificates \
+      cmake \
+      git \
+      g++ \
+      libboost-all-dev \
+      libcgal-dev \
+      libcurl4-gnutls-dev \
+      libgmp-dev \
+      libjson-c-dev \
+      libmpfr-dev \
+      libprotobuf-c-dev \
+      libsqlite3-dev \
+      libtiff-dev \
+      libtool \
+      libxml2-dev \
+      make \
+      pkg-config \
+      postgresql-server-dev-12 \
```

Comment:
suggestion: `postgresql-server-dev-$PG_MAJOR \`
imho: easier to adapt to other pg versions.  like  PG11/PG10

Reply:
Thanks, I fixes to use %%PG_MAJOR%% to replace by sed command.

Reply:
Oh, sorry. I use $PG_MAJOR instead of %%PG_MAJOR%%.

Reply:
I fixed it.